### PR TITLE
Add back button on credit card and billing address screens

### PR DESCRIPTION
**Pull request by yuki24**
finishes https://artsyproduct.atlassian.net/browse/PURCHASE-215

It is barely touchable in Emission since there is a native back button, but in Eigen you should be able to see and tap the button easily. Here is a screenshot that's in slow motion so you can see the button is already there.

back-button

#skip_new_tests

### CODE REVIEW

File: src/lib/Components/Bidding/Screens/BillingAddress.tsx [hunk 1]
```diff
@@ -88,52 +89,56 @@ export class BillingAddress extends React.Component<BillingAddressProps, Billing
...
-              placeholder="Enter your postal code"
-              autoCapitalize="words"
-              {...this.propsForInput("postalCode")}
-            />
-
-            <Button text="Add billing address" onPress={() => this.onSubmit()} />
-          </Container>
-        </ScrollView>
+        <View>
+          <BackButton navigator={this.props.navigator} />
```
Comment (yuki24) [inline src/lib/Components/Bidding/Screens/BillingAddress.tsx]: This looks like a large change, but you change GitHub's "Diff settings" and hide whitespace changes it'll be a lot easier to review:

screen shot 2018-07-02 at 5 41 15 pm

**Comment by yuki24** [comment]
This PR is good for review.

File: src/lib/Components/Bidding/Components/BackButton.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { NavigatorIOS, TouchableWithoutFeedback, ViewProperties } from "react-native"
+
+import { Image } from "../Elements/Image"
+import { theme } from "../Elements/Theme"
+
+interface ContainerWithBackButtonProps extends ViewProperties {
+  navigator: NavigatorIOS
+}
+
+export class BackButton extends React.Component<ContainerWithBackButtonProps> {
```
Comment (sweir27) [inline src/lib/Components/Bidding/Components/BackButton.tsx]: I'm not sure what the convection is in Emission, but this seems like a thing that could live in a more global space eventually.

**Comment by sweir27** [comment]
Nice! @yuki24 this looks good to go. 🎉  Looks like you have a couple of non-trivial conflicts to resolve first, then merge on green 😄

**Pull request merged**